spec/requests/api/v1/people_controller_image_spec.rb

```diff
@@ -13,28 +13,46 @@
         allow(NomisClient::Image).to receive(:get).and_return(nil)
       end
 
-      it 'returns 404' do
-        get "/api/v1/people/#{person.id}/image", params: { access_token: token.token }
+      it 'returns resource not found 404' do
```

Comment:
This was a different test - to check that if the person didn't have an image then we return 404. The non-existant-id case should IMHO be a new test

Reply:
fixed, I've created a different context for that.

Reply:
we still seem to have 2 deletions according to github?